DMTN-248.tex

```diff
@@ -46,8 +46,245 @@
+to compress a visit's worth of alerts, or if the alert distribution latency requirement is relaxed from 1 minute.
+
+\textbf{Recommendation}: Do not compress alert packets, it imposes a risk to rapid time-domain science.
+Potentially reconsider compression if it requires only a few seconds per visit.
+
+
+\section{Multiple packet formats}\label{sec:multiformats}
+
+\textbf{Proposal}: Allow brokers to specify whether their stream should have histories or postage stamps removed.
+
+The main motivation here is not scientific, but to help reduce brokers' processing costs spent removing unneeded information 
+from alert packets. 
+
+There are a few drawbacks to this proposal, but they are mostly technical:
+
+\begin{enumerate}
+\item New scope, as the current plan is for Prompt Processing to create a uniform alert packet.
+\item Streams of full-sized packets could unfairly suffer delays more often (e.g., in crowded fields).
+\item Non-identical packets might cause bookkeeping issues for downstream brokers subscribing to multiple brokers.
+\end{enumerate}
+
+These drawbacks could impose a risk to the particular science goals of brokers using full-sized or non-identical alerts.
+
+\textbf{Recommendation}: None at this time, this is primarily a technical question with little science impact.
+
+
+\section{Pre-filtered streams}\label{sec:prefilter}
+
+\textbf{Proposal}: Allow brokers to request a pre-filtered alert stream, for example to only include alerts in a certain sky 
+region or which meet other criteria (e.g., brightness limits, number of past detections).
+
+As with the multiple packet formats, the main motivation here is not scientific, but to help reduce brokers' processing costs 
+spent recieving unwanted alerts, or to enable Rubin Observatory to distribute alerts to more brokers.
+
+For example, due to the exponential relationship between the number of variable stars and their variability amplitude, and 
+also that of volume and distance modulus, an apparent magnitude limit that is 1 mag brighter than the nominal 5-sigma \texttt{DiaSource} 
+detection limit could reduce the alert stream data rate by 50\%.
+
+However, all seven full-stream brokers have committed to recieving the full stream, and the term downstream broker refers to 
+brokers which will ingest a filtered stream of alerts from the full-stream brokers.
+
+\textbf{Recommendation}: There is no need for Rubin Observatory to provide pre-filtered streams to brokers.
+
+
+\section{When and how to distribute delayed alerts}\label{sec:delayed}
+
+The Data Management System (DMS) is required to support the distribution of at least 40,000 alerts per single standard visit, 
+and for visits producing $\leq$40,000 alerts, no more than 1\% of them fail to have at least 98\% of its alerts distributed 
+within 60 seconds of image readout (based on LSR-REQ-0101 in \citeds{LSE-29}; OSS-REQ-0193 in \citeds{LSE-30}; 
+and DMS-REQ-0392 and -0393 in \citeds{LSE-61}).
+
```

Comment:
Consider separating the above sentence into two - I had to parse it a few times.

Reply:
Separated: "... standard visit. Furthermore, for visits ..."